prover/setup.sh

```diff
@@ -18,7 +18,7 @@ sed -i.backup 's/^proof_loading_mode=.*$/proof_loading_mode="FriProofFromGcs"/'
 rm ../etc/env/base/eth_sender.toml.backup
 sed -i.backup 's/^setup_data_path=.*$/setup_data_path="vk_setup_data_generator_server_fri\/data\/"/' ../etc/env/base/fri_prover.toml
 rm ../etc/env/base/fri_prover.toml.backup
-sed -i.backup 's/^universal_setup_path=.*$/universal_setup_path="..\/keys\/setup\/setup_2^26.key"/' ../etc/env/base/fri_proof_compressor.toml
+sed -i.backup 's/^universal_setup_path=.*$/universal_setup_path="..\/keys\/setup\/setup_2^24.key"/' ../etc/env/base/fri_proof_compressor.toml
```

Comment:
so, now setup.sh will work only for GPU compressors, right? For CPU one would need to change it to use 2^24. I think it's worth mentioning, but not sure where exactly

Reply:
Added info in docs, also updated some things to use 2^24 key. Generally we are able to generate all the things we need with 2^24 key but I'm not fully sure that compressor works with it, will need to test and will add it to my backlog to use only it if so.